api/tests/mocha/services/monitoring.spec.js

```diff
@@ -151,6 +151,7 @@ describe('Monitoring service', () => {
       chai.expect(actual.conflict).to.deep.equal({ count: 40 });
       chai.expect(actual.date.current).to.equal(0);
       chai.expect(actual.replication_limit.count).to.equal(1);
+      chai.expect(actual.connected_users.count).to.equal(1);
```

Comment:
This shouldn't succeed without additional mocking. Extend the mocks so you mock the two calls separately, and return different values so you can assert that the right service is wired up to the right value.